common/deftype/deftype.cpp

```diff
@@ -1998,6 +2049,16 @@ inline void ReleaseAndClear(T * & ptr)
     }
 }
 
+template <class T>
+inline void ReleaseAndClear(std::atomic<T*>& atomicPtr)
```

Comment:
scope: This does not need to be atomic - it is only called single threaded at process termination.